src/gui/command.ts

```diff
@@ -120,36 +120,36 @@ export class FindElementsFileCommand {
                 workspace.workspaceFolders?.map(({ uri }) => uri.fsPath),
             );
         } catch (error: any) {
-            getLog().info(l10n.t("Find element descriptors file: "), error.message || error);
+            getLog().info(l10n.t("Find element descriptor files: "), error.message || error);
         }
         pythonPath = pythonPath || workspace.getConfiguration("python").get("defaultInterpreterPath", "python");
         window.withProgress(
```

Comment:
I don't understand the logic of this piece of code...
Whether we find no element descriptor files, or fail to find one, or...

Reply:
do you mean the with progress section? basically there is a window popping up stating it is finding the visual element descriptor. I can show you in the daily